objects/obj_pause/Draw_77.gml

```diff
@@ -0,0 +1,47 @@
+//Disable alpha blending (Drawing just RGB)
+gpu_set_blendenable(false);

```

Comment:
I don't think this (and the ending set back to true) is needed for our project at this point.

Reply:
I would just comment these out for now as they might be needed later as we add more visuals.

Reply:
Commented out the gpu_set_blendenable(false) and gpu_set_blendenable(true) for now.